widgets/npc/newguy.tw

```diff
@@ -1,64 +1,24 @@
 :: Widget- newguy [widget]
 <<widget newguy>>
 	<<include 'Guy names'>>
-	<<set _age = setup.ageRoll()>>
-	<<set _race = setup.raceRoll()>>
-	<<set _hair = setup.hairRoll(_race, _age, 1)>>
-	<<set _eyes = setup.eyesRoll(_race, _hair)>>
-	<<set _orientation = setup.orientationRoll(1)>>
-	
-	<<set $tmpGuy = {
-		gender: 1,	
-		name: either(_names),
-		beauty: random(20, 70),
-		birthDate: setup.getBirthDate(_age),
-		relationship: 0,
-		corruption: random(0, 10),
-		sub: random(5, 13),
-		happy: random(40, 60),
-		hair: _hair,
-		eyes: _eyes,
-		dick: either('small', 'medium', 'big'),
-		race: _race,
-		location: null,
-		talked: false,
-		food: random(10, 90),
-		strength: random(20, 40),
-		assignedTo: null,	
-		horny: random(0, 30),
-		traits: [],
-		skills: [],
-		likes: [],
-		dislikes: [],
-		personality: setup.personalityTraits(either(1, 2, 2)),
-		endurance: 0,
-	}>>
-	
-<<set $tmpGuy = setup.setSexuality($tmpGuy, _orientation)>>
-<<if ($tmpGuy.likesGuys || $tmpGuy.likesTGirls) && setup.percentageChance(25)>>
-	<<set $tmpGuy.traits = [either('analslut', 'cumslut', 'deepthroat')]>>
-<</if>>
-
-<<if setup.percentageChance(50)>>
-	<<set $tmpGuy.skills = setup.getRandomSkills()>>
-<</if>>
+	<<set _gender = 1>>	
+	<<include 'newnpc'>>
```

Comment:
instead of include I would suggest to create new widget instead for gender as widget argument